container/types.go

```diff
@@ -316,6 +316,7 @@ func (r *PutRequestBody) GetSignature() *refs.Signature {
 
 func (r *PutRequestBody) SetSignature(v *refs.Signature) {
 	if r != nil {
+		v.SetScheme(0)
```

Comment:
Can't we use default scheme constant (see previous comment) from refs package?

Reply:
Since it is a hack to not create `SignatureRFC6979` type, zero is more appropriate here.

Reply:
Got it. Maybe add some issue to do it properly in the future?

Reply:
* https://github.com/nspcc-dev/neofs-api-go/issues/381